### ISSUE TITLE
DL-track

### ISSUE THREAD

**Issue by justheuristic** [goto-ru/Basic_ML#2]
- [x] Линейные модели и градиенты. Feature extraction. "А что если фичи сами себя найдут" Biological inspiration. Unsupervised feature learning. RBM, PCA, всё в scikit-learn
- [x] Intro (s4 theano). Chain rule aka backprop. Nonlinearity. Stochastic gradient descent. Mommentum, rmsprop, adagrad. dl_s4 -> theano, logistic regression, Shallow NN in theano.
- [x] Convolutional NN, pooling, dropout, deep learning @image architectures. dl_s4@mlhep -> lasagne part.
- [x] Project example: cats Vs dogs. Information flow approach. "Fix my poor NN" game. Fine-tuning.
  Moar cats-v-dogs.
- [x] Autoencoders. Shrinking AE. Expanding AE problem. Regularization. Distortion & denoizing AE.
  s8 Smile adjustment
- [x] Generative adversarial networks. Cross-domain learning. s7 Generating NotMnist or Mnist or faces.
- [x] batchnorm из физики. Nonlinearity choice, maxout, batch-norm, parallel architectures.
  cats-vs-dogs from scratch && compare learning rates with/without batchnorm.
- [x] text CNNs, avito [ссылка]
- [x] Recurrent neural networks. Text encoding. Text generation. Language models. dl_s10 -> generate your own Harms/laws/R&P/any
- [ ] conversation model from scratch

**Comment by justheuristic** [comment]
https://github.com/yandexdataschool/HSE_deeplearning
